docs/high/dataset.rst

```diff
@@ -566,6 +567,14 @@ Reference
 
         Integer giving the total number of elements in the dataset.
 
+    .. attribute:: nbytes
+
+        Integer giving the total number of bytes required to load the full dataset into RAM (i.e. `dset[()]`).
+        This may not be the amount of disk space occupied by the dataset,
+        as datasets may be compressed when written or only partly filled with data.
+        Also note that this value does not include the array overhead, as it only describes the size of the data itself.
```

Comment:
```suggestion
        This value also does not include the array overhead, as it only describes the size of the data itself.
```

Something I picked up [from Wikipedia](https://en.wikipedia.org/wiki/Wikipedia:It_should_be_noted) years ago - "note that" can almost always be removed with no change in the meaning.